### DISCUSSION TITLE
v2.17.36

### DISCUSSION THREAD

**Discussion by danharrin** [filamentphp/filament#6419] [General]
## What's Changed
* Revert "Do not send unnecessary headers when pretending to be a file" by @danharrin in https://github.com/filamentphp/filament/pull/6418

**Full Changelog**: https://github.com/filamentphp/filament/compare/v2.17.35...v2.17.36

<hr /><em>This discussion was created from the release <a href='https://github.com/filamentphp/filament/releases/tag/v2.17.36'>v2.17.36</a>.</em>

**Comment by coswat** [comment]
ok
